src/test/java/org/jenkinsci/plugins/compress_artifacts/ZipStorageTest.java

```diff
@@ -290,16 +290,12 @@ public void run() {
         };
 
         compressor.start();
-        try {
-            Thread.sleep(1000);
+        Thread.sleep(200);
 
-            assertTrue(compressor.isAlive());
+        assertTrue(compressor.isAlive());
 
-            assertArrayEquals(new String[0], zs.list());
-            assertArrayEquals(new VirtualFile[0], zs.list("*"));
-        } finally {
-            compressor.stop();
```

Comment:
The automated test that calls `compressor.start()` is now modified to never call `compressor.stop()`  because Java 8 and later have modified the Thread.stop() method to always throw an exception when it is called.  An [Oracle Tecnote](https://docs.oracle.com/javase/8/docs/technotes/guides/concurrency/threadPrimitiveDeprecation.html) describes the details.  I took the simple approach and accepted that if the thread were leaked from a test, it would be a very short-lived leak because the test process would end very soon.